source/administration/changelog.md

```diff
@@ -188,6 +190,8 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 ## Release v5.29 - [Quality Release](https://handbook.mattermost.com/operations/research-and-development/product/release-process/release-overview)
 
+- **v5.29.2, released 2021-01-16**
+  - Fixed an issue where the ``UnreadMentions`` column was missing in the ``ThreadMemberships`` table for servers upgrading from an earlier version. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.29.2 or later.
```

Comment:
...was missing in the ``ThreadMemberships`` table for servers upgrading from v5.29.0